src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
+
+    virt_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Virtualization type """
+
+    accounts = attr.ib(type=str, default=None, validator=optional_str)
+    """Accounts """
+
+    groups = attr.ib(type=str, default=None, validator=optional_str)
+    """Groups"""
```

Comment:
@jajreidy @lslebodn Will it be required to use multiple accounts/groups or just the internal one?

Because looking here in [pubtools-ami](https://github.com/release-engineering/pubtools-ami/blob/main/pubtools/_ami/tasks/push.py#L422-L430) they get it from command line args so maybe we can get this from the credentials file as well, instead of relying on StArMap and adding an attribute that might not be required?

Then we could have the credentials file (from args) more or less like this:

```
{
    "marketplace_account": "aws-na",
    "auth": {
        "access_id": "**************",
        "secret_key": "**************",
        "region": "**************",
        "groups": ["group1", "group2", ...],
    },
}
```

WDYT?

Reply:
Yeah that makes sense.

Reply:
In this case I am removing this metadata for now, in case we need this one in future, we can add it.